File: c+1/starting/SpeedyBoi/Comp.java
import java.util.HashSet;

class Comp{

    public static void main(String[] args) {
        HashSet<Integer> hash = new HashSet<Integer>();
        for (int i = 0; i < 1000000; i ++) {
            hash.add(i);
        }
        for (int i = 0; i < 1000000; i++) {
            hash.contains((Integer)i);
        }
        for (int i = 0; i< 1000000; i++) {
            hash.remove((Integer) i);
        }
        System.out.println("Done");
    }


}

File: c+1/starting/SpeedyBoi/Comp.py

def main():
    hash = set()
    for i in range(1000000):
        hash.add(i)
    for i in range(1000000):
        x = i in hash
    for i in range(1000000):
        hash.remove(i)
    print("Done")


if __name__ == '__main__':
    main()
